src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -128,19 +155,58 @@ staticField java.util.Locale SIMPLIFIED_CHINESE
 staticField java.util.Locale TRADITIONAL_CHINESE
 staticField java.util.Locale UK
 staticField java.util.Locale US
+method java.util.Map clear
 method java.util.Map containsKey java.lang.Object
+method java.util.Map entrySet
 method java.util.Map get java.lang.Object
 method java.util.Map keySet
 method java.util.Map put java.lang.Object java.lang.Object
 method java.util.Map putAll java.util.Map
+method java.util.Map size
 method java.util.Map values
 method java.util.Map$Entry getKey
 method java.util.Map$Entry getValue
 staticMethod java.util.TimeZone getTimeZone java.lang.String
+new java.util.TreeSet
 staticMethod java.util.UUID randomUUID
 method java.util.concurrent.Callable call
+method java.util.regex.MatchResult group int
+method java.util.regex.Matcher matches
 method java.util.regex.Matcher replaceAll java.lang.String
 method java.util.regex.Matcher replaceFirst java.lang.String
+method javax.xml.parsers.DocumentBuilder parse org.xml.sax.InputSource
```

Comment:
This might be dangerous. IIRC it allows any URL or whatever.

I think it would be better to encapsulate things like XML or JSON parsing into the Pipeline Utility Steps plugin, where you could limit it to a `String` input (for example from `readFile`).